cosmos-auth/conf/cosmos-auth.json

```diff
@@ -1,6 +1,8 @@
 {
```

Comment:
Maybe it is a good idea to provide documentation about this file and the configuration parameters it includes in the README.md file, what do you think? :)